### PR TITLE
Fix warnings and add javadoc

### PR DESCRIPTION
**Pull request by 1239uth**

### CODE REVIEW

File: app/src/main/java/com/example/fitappa/Model/Entity/User.java [hunk 1]
```diff
@@ -14,16 +13,14 @@
...
-    public User(String email, String username, String password) {
+    public User(String email, String username) {
         this.username = username;
-        this.password = password;
         this.email = email;
     }
 
-    public User(String email, String username, String password, String uniqueID) {
-        this(email, username, password);
+    public User(String email, String username, String uniqueID) {
```
Comment (nxabdullah) [inline app/src/main/java/com/example/fitappa/Model/Entity/User.java:22]: Javadoc?

**Review by nxabdullah — CHANGES_REQUESTED** [review]
Potential Javadoc where I left comment

File: app/src/main/java/com/example/fitappa/Model/Entity/User.java (hunk 1 above)
Comment (1239uth) [reply app/src/main/java/com/example/fitappa/Model/Entity/User.java:30]: Fixed now.

**Review by sourenrex — APPROVED** [review]
Looks good to me I don't see any issues

**Pull request merged**